### ISSUE TITLE
The weights for StableNormal-turbo

### ISSUE THREAD

**Issue by BIT-DYN** [Stable-X/StableNormal#30]
Such an amazing job!

I have downloaded the original weights for StableNormal followed [#14](https://github.com/Stable-X/StableNormal/issues/14#issuecomment-2345283779).

But where can I download the weights file for StableNormal-turbo?
I want to test which of the two is better for our scenes. 
Thanks!

**Comment by hugoycj** [comment]
We are currently conducting beta testing for StableNormal_turbo to optimize the balance between sharpness and accuracy. You can try the demo on Hugging Face Spaces at huggingface.co/spaces/Stable-X/StableNormal_turbo_beta.
For local testing, please use the following commands:
```
import torch
from PIL import Image

# Load an image
input_image = Image.open("path/to/your/image.jpg")

# Create predictor instance
predictor = torch.hub.load("hugoycj/StableNormal", "StableNormal_turbo", trust_repo=True, yoso_version='yoso-normal-v1-5')

# Generate normal map using alpha channel for masking
normal_map = predictor(rgba_image, data_type="object")  # Will mask out background, if alpha channel is avalible, else use birefnet
normal_map = predictor(rgba_image, data_type="outdoor")  # Will use Mask2Former to mask out sky and plants
normal_map = predictor(rgba_image, data_type="indoor") # Will not mask out

# Apply the model to the image
normal_image = predictor(input_image)

# Save or display the result
normal_image.save("output/normal_map.png")
```

**Comment by BIT-DYN** [comment]
Thank you for your quick reply!

Since my terminal doesn't have direct access to huggingface (but the browser does).
So I chose to download the [v1-5 weights ](https://huggingface.co/Stable-X/yoso-normal-v1-5) locally, but there seems to be some problems.

![image](https://github.com/user-attachments/assets/c2bb90f4-2d59-401d-b8e0-c8f2139a2071)

 
[v1-0  weights ](https://huggingface.co/Stable-X/yoso-normal-v1-5) is available, but the results are very unsatisfactory.

![e0ae8d822f1756bda926e631ce59ebe](https://github.com/user-attachments/assets/63b9d609-afe6-4acd-923b-c75a41096041)


Do you have any suggestions?

**Comment by xingchen2022** [comment]
> Thank you for your quick reply!
> 
> Since my terminal doesn't have direct access to huggingface (but the browser does). So I chose to download the [v1-5 weights ](https://huggingface.co/Stable-X/yoso-normal-v1-5) locally, but there seems to be some problems.
> 
> ![image](https://private-user-images.githubusercontent.com/74786439/393725228-c2bb90f4-2d59-401d-b8e0-c8f2139a2071.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQzNzE2NDYsIm5iZiI6MTczNDM3MTM0NiwicGF0aCI6Ii83NDc4NjQzOS8zOTM3MjUyMjgtYzJiYjkwZjQtMmQ1OS00MDFkLWI4ZTAtYzhmMjEzOWEyMDcxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjE2VDE3NDkwNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE5ODk4MjI0N2IwZTAwZGNkYjA0NGViOTJhZmExM2FkMzYwM2VlYjRkZTViZjhhOTFjMWFjNzM2MzZkY2Q5ODQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.f7aYfnabiWnrmp6cmsA548xmlQAvxfxInTomatbnKew)
> 
> [v1-0 weights ](https://huggingface.co/Stable-X/yoso-normal-v1-5) is available, but the results are very unsatisfactory.
> 
> ![e0ae8d822f1756bda926e631ce59ebe](https://private-user-images.githubusercontent.com/74786439/393725285-63b9d609-afe6-4acd-923b-c75a41096041.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQzNzE2NDYsIm5iZiI6MTczNDM3MTM0NiwicGF0aCI6Ii83NDc4NjQzOS8zOTM3MjUyODUtNjNiOWQ2MDktYWZlNi00YWNkLTkyM2ItYzc1YTQxMDk2MDQxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjE2VDE3NDkwNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTllZDk2MDlmZjBhMjkzZjU2OTBiZGEyMDJhMWU2Njk5Njc3YmY2N2MzZDMyNjUwOWZiM2ExZTc2ZmRlNjhlMjgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.a7t0W8hwp0ndjTaPhpjMpgQTacJhAZpZqz6EmLhRRrQ)
> 
> Do you have any suggestions?

same question, have you solved this？

**Comment by xingchen2022** [comment]
@BIT-DYN hi, have you find the proper way to reproduce the result like online demo？I have tried different weights in #31

**Comment by BIT-DYN** [comment]
> @BIT-DYN hi, have you find the proper way to reproduce the result like online demo？I have tried different weights in #31

Unfortunately, I have not solved this problem so far.

**Comment by hugoycj** [comment]
@BIT-DYN Sorry for the late reply. Could you share your input image and the normal result from the HuggingFace demo? StableNormal may fail for defocused, horizontally oriented or cartoon-style images. It might be our model's problem. Thanks.

**Comment by YouJiacheng** [comment]
friendly ping @hugoycj 
+1 can't run v1-5 or v1-8-1 due to missing modules.
v1-0 result is very blurry.
ModelScope demo seems to call aliyun DashScope API...
